scenario/LiquidationBotScenario.ts

```diff
@@ -443,7 +443,7 @@ scenario(
     },
     cometBalances: {
       albert: {
-        $asset0: ' == 5',
+        $asset0: ' == 10',
```

Comment:
Due to prices falling, we were apparently not borrowing enough anymore (below min)